test_utils/test_utils.h

```diff
@@ -35,5 +38,7 @@ struct test_list_t
 };
 
 int get_test_set(int *, int, const char *, struct test_list_t *);
+void fill_with_pseudorandom_data(void* buffer, size_t size);
+void fill_with_pseudorandom_data_seed(uint64_t seed, void* buffer, size_t size);
```

Comment:
If this isn't used outside of test_utils.c, it should be private to that file.

Reply:
I was originally using it outside of the file but it's no longer the case. Will update the pr to make it static.